code/data_preprocessing/phenotype/gene_annotation.ipynb

```diff
@@ -94,10 +94,10 @@
    "source": [
     "sos run gene_annotation.ipynb annotate_coord \\\n",
     "    --cwd output \\\n",
-    "    --phenoFile data/... \\\n",
-    "    --annotation-gtf reference_data/Homo_sapiens.GRCh38.103.chr.reformatted.gene.ERCC.gtf  \\\n",
+    "    --phenoFile data/MWE.pheno_log2cpm.tsv \\\n",
+    "    --annotation-gtf data/gene.gtf  \\\n",
     "    --sample-participant-lookup data/sampleSheetAfterQC.txt \\\n",
-    "    --container container/rna_quantification.sif \\"
+    "    --container ./rna_quantification.sif --phenotype_id_type gene_name"
```

Comment:
you might have noticed my convention to use eg `--phenotype-id-type` and not `--phenotype_id_type` in command calls. Please stick to my convention to be consistent.